bin/modules/beer_agent/commands.dart

```diff
@@ -0,0 +1,176 @@
+        ..appendBold(saleSize)
+        ..append(' new beers!')
+        ..appendNewLine()
+        ..appendNewLine()
+        ..append('Some of them are:')
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[0].name)
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[1].name)
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[2].name)
+        ..appendNewLine()
+        ..appendNewLine();
+    } else if (saleSize == 2) {
+      oelMessage
+        ..append(':beer: ')
+        ..appendBold(saleDate)
+        ..appendNewLine()
+        ..append('This release has ')
+        ..appendBold(saleSize)
+        ..append(' new beers!')
+        ..appendNewLine()
+        ..appendNewLine()
+        ..append('Some of them are:')
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[0].name)
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[1].name)
+        ..appendNewLine()
+        ..appendNewLine();
+    } else if (saleSize == 1) {
+      oelMessage
+        ..append(':beer: ')
+        ..appendBold(saleDate)
+        ..appendNewLine()
+        ..append('This release has ')
+        ..appendBold(saleSize)
+        ..append(' new beer!')
+        ..appendNewLine()
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[0].name)
+        ..appendNewLine()
+        ..appendNewLine();
+    }
+  }
+
+  oelMessage
+    ..append('---')
+    ..appendNewLine()
+    ..append('For more information: https://systembevakningsagenten.se/')
+    ..appendNewLine()
+    ..appendNewLine()
+    ..append('Cheers! :beers:');
+
+  //Send message
+  await ctx.respond(oelMessage);
+}
+
+Future<void> _releaseCommand(ISlashCommandInteractionEvent ctx) async {
+  var input = ctx.args;
+  if (input.length == 1) {
+    var parsedDate = DateTime.tryParse(input[0].value);
+
+    if (parsedDate != null) {
+      await BeerAgentModule()._updateBeerSales();
+      for (var sale in BeerAgentModule().beerSales) {
+        var saleDate = DateTime.parse(sale.saleDate);
+        if (parsedDate == saleDate) {
+          //Compile beer list to string and sort by name.
+          var beerStr = '';
+          sale.beerList.sort((a, b) => a.name.compareTo(b.name));
+          sale.beerList.forEach((element) {
+            beerStr += '- ' + element.name + '\n';
+          });
+
+          //Bulild reply
+          var slappMessage = MessageBuilder()
+            ..append(ctx.interaction.userAuthor!.mention)
+            ..appendNewLine()
+            ..append(' :beers: ')
+            ..appendBold(input[0].value)
+            ..appendNewLine()
+            ..append('Innehåller ')
+            ..appendBold(sale.beerList.length)
+            ..append(' nya öl:')
+            ..appendNewLine()
+            ..appendNewLine()
+            ..append(beerStr);
```

Comment:
Would be nice if we could try to extract the MessageBuilder logic overall to avoid clutter.